pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java

```diff
@@ -69,17 +69,24 @@ public LocalBookkeeperEnsemble(int numberOfBookies, int zkPort, int bkBasePort)
 
     public LocalBookkeeperEnsemble(int numberOfBookies, int zkPort, int bkBasePort, String zkDataDirName,
             String bkDataDirName, boolean clearOldData) {
+        this(numberOfBookies, zkPort, bkBasePort, zkDataDirName, bkDataDirName, clearOldData, null);
+    }
+
+    public LocalBookkeeperEnsemble(int numberOfBookies, int zkPort, int bkBasePort, String zkDataDirName,
+            String bkDataDirName, boolean clearOldData, String advertisedAddress) {
         this.numberOfBookies = numberOfBookies;
         this.HOSTPORT = "127.0.0.1:" + zkPort;
         this.ZooKeeperDefaultPort = zkPort;
         this.initialPort = bkBasePort;
         this.zkDataDirName = zkDataDirName;
         this.bkDataDirName = bkDataDirName;
         this.clearOldData = clearOldData;
-        LOG.info("Running " + this.numberOfBookies + " bookie(s).");
+        this.advertisedAddress = null == advertisedAddress ? "127.0.0.1" : advertisedAddress;
```

Comment:
should we use `ServiceConfigurationUtils.getDefaultOrConfiguredAddress(advertisedAddress)`?

Reply:
@rdhabalia we can't use that class, because this change is in pulsar-zookeeper-utils while `ServiceConfigurationUtils` is in broker module.

Reply:
yes..that's true. should we use `InetAddress.getLocalHost().getHostName()` instead `127.0.0.1`.?